app/serializers/page_category_serializer.rb

```diff
@@ -16,6 +16,10 @@ def description
   end
 
   def type
-    'guide'
+    guide || object.layout.identifier
+  end
+
+  def guide
+    {'article' => 'guide'}[object.layout.identifier]
```

Comment:
Exactly, just return `'guide'` as there is no other option here, well, other than if you index into this hash with a non-existing key from `object.layout.identifier`. Or am I missing something obvious?